sufia-models/app/services/sufia/generic_file_indexing_service.rb

```diff
@@ -6,6 +6,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('file_format')] = object.file_format
         solr_doc[Solrizer.solr_name('file_format', :facetable)] = object.file_format
         solr_doc['all_text_timv'] = object.full_text.content
+        solr_doc[Solrizer.solr_name('file_size', Solrizer::Descriptor.new(:integer, :stored);)] = object.content.size.to_i
```

Comment:
----- Original Message -----
> From: "Nathan Rogers" <notifications@github.com>
> To: "projecthydra/sufia" <sufia@noreply.github.com>
> Cc: "Michael J. Giarlo" <mjg36@psu.edu>
> Sent: Thursday, June 18, 2015 8:43:28 AM
> Subject: Re: [sufia] Support collection size calculations using a Solr query instead of lo… (#1204)
> 
> > @@ -6,6 +6,7 @@ def generate_solr_document
> >          solr_doc[Solrizer.solr_name('file_format')] = object.file_format
> >          solr_doc[Solrizer.solr_name('file_format', :facetable)] =
> >          object.file_format
> >          solr_doc['all_text_timv'] = object.full_text.content
> > +        solr_doc[Solrizer.solr_name('file_size',
> > Solrizer::Descriptor.new(:integer, :stored);)] = object.content.size.to_i
> 
> The semi colon has been eradicated. Regarding the constant is your concern
> over performance or just aesthetics? Until STORED_INTEGER_DESCRIPTOR is used
> in other places the performance difference is neglible as can be seen if you
> run the following script in a Pry console

Not performance or aesthetics, but maintainability -- since both are relying on the same descriptor, they should share a definition so that it can be updated in one place when we're ready to change it.